### PR TITLE
Comment line "wav = np.concatenate([wav] * 8, axis=1)" In the ONNX inference script

### PR DESCRIPTION
**Pull request by jiayukk**
In the ONNX inference script, there is a line wav = np.concatenate([wav] * 8, axis=1), which causes the output waveform to become eight times the length of the input audio. Commenting this line out fixes the issue.

### CODE REVIEW

**Comment by aask1357** [comment]
Hello! Thank you for the PR.

Originally, the concatenation was intentional. Since the default test audio (`onnx/p232_013.wav`) is quite short (3.9s), we increased the input length to ensure that RTF measurements are stable.

However, I agree that 8x duplication is problematic, especially when users use their own `--audio-path`, other than using a default `--audio-path`.

I propose replacing the default test audio with a longer file (around 30s). This will allow for stable RTF measurement. Also, we should remove the concatenation lines from all related scripts in the `scripts` folder (`export_onnx_spec.py`, `export_onnx.py`, `test_onnx_spec.py`, and `test_onnx.py`).

What do you think about this approach?

**Comment by jiayukk** [comment]
I agree with you. For the output, we can directly crop it to the original input length to ensure correctness. Record the original input length before concatenation, when saving the output audio, crop back to the original length.

**Comment by jiayukk** [comment]
I noticed that in your model, only the middle enhancement layer performs inter-frame processing. Therefore, I compared your FastEnhancer model trained on the DNS dataset and found that it does not handle noise that persists in the frequency domain very well (only the Large version performs well). In contrast, models that use 2D convolutions with a temporal kernel size greater than 1 in both the encoder and the decoder tend to perform better in such cases.

**Comment by aask1357** [comment]
Thank you for your contributions!
Regarding the inter-frame processing in the encoder and decoder, I will train a model with a time-axis kernel size of 3 and calculate objective measures for DNS-Challenge dataset.

**Comment by jiayukk** [comment]
Thank you for your experiments. The fact that the metrics only show a small improvement is fair since this type of noise is usually from music or juicers. Under the small RTF, following the paper and using 1D convolution is a more cost-effective approach.

**Comment by aask1357** [comment]
I see. In that case, I think increasing the proportion of music and juicer noise in the training data might help.
But it is really interesting that the default versions fail to handle music and juicer noise while Conv2d models are more robust to them. Thank you for sharing your findings!
